index.js

```diff
@@ -286,8 +287,8 @@ exports.build = function build(argv, callback) {
     runWait('npm --quiet pack', function(er, output) {
       if (er) return callback(er);
 
-      // npm pack output is a single line with the pack file name
-      var src = output.split('\n')[0];
+      var pkg = require(path.resolve('package.json'));
```

Comment:
Do we make any modifications to package.json? I don't think so.. but there should be a comment here pointing out that it this line will add it as-is to the `require.cache` just so there's a clue for future us when we're trying to figure out why it's not "loading the right file".